test/test_quantized.py

```diff
@@ -532,10 +532,9 @@ def equal_ref(X, params, X_scheme, X2, params2, X2_scheme):
 
 
 @unittest.skipIf(
-    TEST_WITH_UBSAN or not torch.fbgemm_is_cpu_supported(),
-    " Quantized Linear requires FBGEMM. FBGEMM does not play"
-    " well with UBSAN at the moment, so we skip the test if"
-    " we are in a UBSAN environment.",
+    not torch.fbgemm_is_cpu_supported(),
```

Comment:
Remove TEST_WITH_UBSAN import at the top.

Reply:
QNNPACK test also used `TEST_WITH_UBSAN` to disable UBSAN test. Let me check if we can also remove that.

Reply:
We don't want to remove it for QNNPACK just yet. Some kernels still fail with ASAN and have to be fixed.

Reply:
Will keep the `TEST_WITH_UBSAN` for QNNPACK